<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bopmgUICharWindow</class>
 <widget class="QWidget" name="bopmgUICharWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI-Characteristics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_connection">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_status">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string>not connected</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cob_ports"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_connect">
        <property name="text">
         <string>connect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_resetInfo">
        <property name="text">
         <string>reset / refresh</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_values">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="1">
       <widget class="QCheckBox" name="chb_setZeroWhenFinished">
        <property name="text">
         <string>set 0 when finished</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QFrame" name="frame_setValues">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_setValue_lbl">
           <property name="text">
            <string>set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="5">
          <widget class="QComboBox" name="cob_setValue"/>
         </item>
         <item row="0" column="6" colspan="2">
          <widget class="QLabel" name="lbl_stepSize_lbl">
           <property name="text">
            <string>step size:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8" colspan="2">
          <widget class="QDoubleSpinBox" name="dsb_stepSize">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="11">
          <widget class="QCheckBox" name="chb_calcValues">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>calc values</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_setFrom_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>from</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QDoubleSpinBox" name="dsb_fromValue">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="lbl_to_lbl">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5" colspan="3">
          <widget class="QDoubleSpinBox" name="dsb_toValue">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="8" colspan="2">
          <widget class="QComboBox" name="cob_unit"/>
         </item>
         <item row="1" column="10" colspan="2">
          <widget class="QLabel" name="lbl_unit_lbl">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>(unit for all fields)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="lbl_repeat_lbl">
           <property name="text">
            <string>repeat</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spb_repeat">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="4">
          <widget class="QCheckBox" name="chb_loop">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>roundly</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="7" colspan="2">
          <widget class="QLabel" name="lbl_ticksLeft_lbl">
           <property name="text">
            <string>ticks left:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="9" colspan="2">
          <widget class="QLabel" name="lbl_ticksApprox">
           <property name="text">
            <string>(ca.)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="11">
          <widget class="QLabel" name="lbl_ticksLeft">
           <property name="text">
            <string>10000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>164</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QFrame" name="frame_current">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_current_lbl">
           <property name="text">
            <string>current:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_current">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="chb_shareCurrent">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>share</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QFrame" name="frame_voltage">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_voltage_lbl">
           <property name="text">
            <string>voltage:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_voltage">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="chb_shareVoltage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>share</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_startStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
